Doc/library/tarfile.rst

```diff
@@ -100,7 +100,10 @@ Some facts and figures:
 
    For modes ``'w:gz'``, ``'r:gz'``, ``'w:bz2'``, ``'r:bz2'``, ``'x:gz'``,
    ``'x:bz2'``, :func:`tarfile.open` accepts the keyword argument
-   *compresslevel* (default ``9``) to specify the compression level of the file.
+   *compresslevel* to specify the compression level of the file.
+   This defaults to ``9`` for gzip and bz2 compression and to
+   :class:`PRESET_DEFAULT <lzma.LZMACompressor>` for lzma compression.
```

Comment:
This sounds like you can get LZMA compression with the _gz_ or _bz2_ modes, which doesn’t seem sensible.

Reply:
Thanks, I've re-worded this.

Reply:
Perhaps add _w:xz_ etc to the list of modes that _compresslevel_ applies to?

Reply:
I decided I was wrong about this and substantially changed course - your comments showed me the error of my ways. Thanks!